neusomatic/python/call.py

```diff
@@ -149,9 +149,11 @@ def pred_vcf_records_path((path, true_path_, pred_all, chroms, vartype_classes,
             if type_pred != "INS" and center_ in zref_pos:
                 if center in nzref_pos:
                     center_ = center
-                else:
+                elif len(nzref_pos) > 0:
```

Comment:
can just say `elif nzref_pos:` since empty lists evaluate to False and True otherwise.